include/RAJA/pattern/forall.hpp

```diff
@@ -168,13 +168,10 @@ RAJA_INLINE concepts::enable_if<
     type_traits::is_range<Container>>
 forall(ExecutionPolicy&& p, Container&& c, LoopBody&& loop_body)
 {
-
-  using RAJA::internal::trigger_updates_before;
-  auto body = trigger_updates_before(loop_body);
-
+  RAJA_FORCEINLINE_RECURSIVE
```

Comment:
I moved force inline recursive here, is that reasonable or does it need to be at a higher level?

Reply:
@ajkunen has more experience with the use of that macro for the intel compiler. He should probably weigh in.

Reply:
@MrBurmark this makes sense... it isn't really the inlining of the loop_body that has been an issue, but stuff inside of forall_impl (like the iterators, etc.)

Reply:
@ajkunen thanks for commenting.